### PR TITLE
Pigs and speech bubbles 🐽 💬 

### PR DESCRIPTION
**Pull request by ethteck**
🐽 💬

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 29 functions (+0.21%) on `us`.

⚠️ This PR introduces 2 warnings:
- src/os/nusys/nuGfxInit.c:13: warning: initialization from incompatible pointer type

File: include/npc.h [hunk 1]
```diff
@@ -323,7 +323,7 @@ typedef struct Enemy {
     /* 0xB4 */ s8 aiPaused;
     /* 0xB5 */ s8 unk_B5;
     /* 0xB6 */ char unk_B6[2];
-    /* 0xB8 */ EvtScript* unk_B8; // some bytecode
+    /* 0xB8 */ EvtScript* rot; // some bytecode
```
Comment (nanaian) [inline include/npc.h:326]: Remove comment
Also why is this rot? A rotation script?? Looks like a find replace accident considering the rest of the pr

File: src/audio.h [hunk 2]
```diff
@@ -550,7 +550,7 @@ typedef struct SoundManager {
     /* 0x08E */ char unk_8E[0x2];
     /* 0x090 */ SoundManager90 unk_90[4];
     /* 0x0A0 */ SoundManagerA0 unk_A0[4];
-    /* 0x0B8 */ u16 unk_B8;
+    /* 0x0B8 */ u16 rot;
```
Comment (nanaian) [inline src/audio.h:553]: Find replace accident source?

File: src/e20eb0.c [hunk 3]
```diff
@@ -4,8 +4,157 @@
...
+                playerStatus->animFlags &= ~PLAYER_STATUS_ANIM_FLAGS_SPEECH_PROMPT_AVAILABLE;
+                return;
+            }
+            var_a2 = e20eb0_UnkAngleFunc1(unk10);
+            break;
+    }
+
+    D_802B79A8_E21858->unk_2B = var_a2;
+    if (playerStatus->encounteredNPC != NULL) {
+        if (playerStatus->encounteredNPC != D_802B79A8_E21858->unk_00) {
```
Comment (nanaian) [inline src/e20eb0.c:150]: Can type this D_xxx as Npc*
if you dont want to do the replace maybe throw a comment on it

File: src/world/script_api/7E0E80.c [hunk 4]
```diff
@@ -885,21 +880,17 @@ ApiStatus MakeShop(Evt* script, s32 isInitialCall) {
     shop->inventoryItemFlags = inventoryItemFlags;
 
     numShopItems = 0;
-    items = inventory->itemID;
-    while (items != 0) {
+    while (inventory->itemID != 0) {
```
Comment (nanaian) [inline src/world/script_api/7E0E80.c:883]: Iirc theres ITEM_NONE?

File: src/world/script_api/7E3700.c [hunk 5]
```diff
@@ -109,11 +109,104 @@ ApiStatus func_80282C40(Evt* script, s32 isInitialCall) {
...
+    yThing = gPlayerStatus.position.y;
+    zThing = gPlayerStatus.position.z;
+
+    xThing = xThing - x;
+    yThing = yThing - y;
+    zThing = zThing - z;
+    if (xThing < 0) {
+        do {
+        xThing -= 25;
+        } while (0);
```
Comment (nanaian) [inline src/world/script_api/7E3700.c:144]: Bad indentation and empty line

**Review by nanaian — CHANGES_REQUESTED** [review]

**Pull request merged**